assets/airports/eddl.json

```diff
@@ -0,0 +1,610 @@
+                "23R": [["DL237", "A6+"], "LMA"]
+            },
+            "body": [],
+            "exitPoints": {
+                "NETEX": ["NETEX"]
+            },
+            "draw": [
+                ["DL050", "DL053", "NIKOG", "ORSOV", "VEBAK", "NETEX*"],
+                ["DL236", "LMA", "NETEX"],
+                ["DL237", "LMA", "NETEX"]
+            ]
+        },
+        "NUDGO": {
+            "icao": "NUDGO",
+            "name": "NUDGO",
+            "suffix": {"05L":"3J", "05R":"3Z", "23L":"4T", "23R":"2H"},
+            "rwy": {
+                "05L": ["DL050", "DL058", "BAM", "ANAVI"],
+                "05R": ["DL050", "DL058", "BAM", "ANAVI"],
+                "23L": ["DL239", "DL241", "ELBAL"],
+                "23R": ["DL240", "DL241", "ELBAL"]
+            },
+            "body": ["MAMIB", "KULIK"],
+            "exitPoints": {
+                "NUDGO": ["NUDGO"]
+            },
+            "draw": [
+                ["DL050", "DL058", "BAM", "ANAVI","MAMIB","KULIK","NUDGO*"],
+                ["DL239", "DL241", "ELBAL", "MAMIB"],
+                ["DL240", "DL241"]
+            ]
+        },
+        "NVO": {
+            "icao": "NVO",
+            "name": "Noervenich",
+            "suffix": {"05L":"1Y", "05R":"1Z", "23L":"1T", "23R":"1U"},
+            "rwy": {
+                "05L": ["DL050", "DL058", "BAM"],
+                "05R": ["DL050", "DL058", "BAM"],
+                "23L": ["DL239", "DL241"],
+                "23R": ["DL240", "DL241"]
+            },
+            "body": [],
+            "exitPoints": {
+                "NVO": ["NVO"]
+            },
+            "draw": [
+                ["DL050", "DL058", "BAM", "NVO*"],
+                ["DL239", "DL241", "BAM"],
+                ["DL240", "DL241"]
+            ]
+        },
+        "SONEB": {
+            "icao": "SONEB",
+            "name": "SONEB",
+            "suffix": {"05L":"4J", "05R":"4Z", "23L":"6T", "23R":"2H"},
+            "rwy": {
+                "05L": ["DL050", "DL053", "NIKOG", "LUSIX", ["DL054", "A140+"]],
+                "05R": ["DL050", "DL053", "NIKOG", "LUSIX", ["DL054", "A140+"]],
+                "23L": ["DL230", ["DL246", "S190-"], "DL248", "ERKUM", ["DL245", "A140+"]],
+                "23R": ["DL231", ["DL246", "S190-"], "DL248", "ERKUM", ["DL245", "A140+"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "SONEB": ["SONEB"]
+            },
+            "draw": [
+                ["DL050", "DL053", "NIKOG", "LUSIX", "DL054", "SONEB*"],
+                ["DL230", "DL246", "DL248", "ERKUM", "SONEB"],
+                ["DL231", "DL246"]
+            ]
+        }
+    },
+    "stars": {
+        "BIKMU": {
+            "icao": "BIKMU",
+            "name": "BIKMU",
+            "suffix": {"05L":"1X", "23R":"1G"},
+            "entryPoints": {
+                "BIKMU": ["BIKMU"]
+            },
+            "body": [
+                [""]
```

Comment:
I guess (although I'm not a json expert) that this has one empty string entry rather than an empty array.